src/zui/ZUIPersonGridEditCell.tsx

```diff
@@ -195,12 +198,20 @@ const ZUIPersonGridEditCell: FC<{
                       </List>
                     )}
                     {searching && (
-                      <List {...autoComplete.getListboxProps()}>
-                        <ListSubheader sx={{ position: 'relative' }}>
-                          {suggestedPeople.length
-                            ? messages.personGridEditCell.otherPeople()
-                            : messages.personGridEditCell.searchResults()}
-                        </ListSubheader>
+                      <List
+                        {...autoComplete.getListboxProps()}
+                        subheader={
+                          <ListSubheader sx={{ position: 'relative' }}>
+                            {suggestedPeople.length >= 0 &&
+                              messages.personGridEditCell.searchResults()}
+                          </ListSubheader>
+                        }
+                      >
+                        {searchResults.length == 0 && (
+                          <CircularProgress
```

Comment:
I think the criteria for when it shows the CircularProgress can be improved. With this criteria, it does not differentiate between "no search results" and "loading/searching" and i felt that it was a bit misleading, since the spinner usually means "wait a second, things will appear here". But if there are no search results it will just spin forever. 
Could there be a way to differentiate between "i have stuff that's gonna appear here" and "no search results"? 
![bild](https://user-images.githubusercontent.com/58265097/227642755-30d4c754-98a3-424b-8013-90d9769f33ea.png)

Reply:
You are right! I´ve added a message to make clear when there are no results!